### PR TITLE
Setup FAQ for linking

### PR DESCRIPTION
**Pull request by saikatbhadra**
- Sets up id for each FAQ question so #hyperlinks work. Sets the id on the `<dt>` but perhaps we should do it on the `<dd>`?
- Adds analytic event when a user clicks on a link to a question or expands a question

### CODE REVIEW

**Comment by saikatbhadra** [comment]
Closes: https://github.com/findthemasks/findthemasks/issues/740

File: views/faq.handlebars [hunk 1]
```diff
@@ -233,11 +230,10 @@
           </dd>
         </dl>
       </section>
-
       <section id="section-makers">
         <h4 class="i18n faq-header" data-i18n="ftm-faq-makerspace-header">Makerspace Community FAQs</h4>
```
Comment (linderr) [inline views/faq.handlebars:234]: Can we adopt this for the header class?

**Comment by linderr** [comment]
This is very cool. Thanks for doing this @saikatbhadra ! I see you enabling us to link to exact questions.
Would it be possible to adopt this for the header class? We also want to link people sections. 
Ideally for the Makers it would be https://findthemakers.com/makers/faq#ftm-faq-makerspace-header

**Comment by saikatbhadra** [comment]
Any id works on the html. Looking at the makers section it looks like it currently has an id of `section-makers` so if you link to https://findthemasks.com/faq#section-makers then it will go to the makers FAQs.

**Comment by linderr** [comment]
That’s great! If you have time to write a Wiki for linking using that for the non-devs like me to refer to when interacting with collaborators/partners or on socials. I can also work on that later tonight & ask for your help to edit it?

**Comment by saikatbhadra** [comment]
In this case, we aren't doing anything special, we are just following standard html method for how to link to different parts of a page. Here are two articles that explain how it works, you could link to them on the wiki:

https://blog.hubspot.com/marketing/jump-link-same-page
https://stackoverflow.com/questions/2835140/how-do-i-link-to-part-of-a-page-hash

**Pull request merged**